tests/features/ConfigurationServiceScalecube/errors.feature

```diff
@@ -0,0 +1,122 @@
+    Given configurationServiceScalecube with following <methods>
+          |<methods>       |
+          |readEntry       |
+          |createEntry     |
+          |readEntryHistory|
+    And   configurationServiceScalecube is created with a valid token
+    And   an existing repository
+    When  user calls one of the methods by providing repository name and without any key
+    Then  user receives an error that the key is not provided
+
+Scenario: Call deleteEntry(), readEntry()  with unexisting key should return 'Configuration repository key not found' error
+    Given configurationServiceScalecube with following <methods>
+          |<methods>        |
+          |deleteEntry      |
+          |readEntry        |
+          |readEntryHistory |
+    And   configurationServiceScalecube is created with a valid token
+    And   an existing repository
+    When  user calls one of the methods by providing an unexisting key
+    Then  user receives an error 'Configuration repository key not found'
+
+Scenario: deleteEntry() without key should delete configuration repository
+          Calling readList method afterwards will throw an error
+    Given configurationServiceScalecube with deleteEntry method
+    And   configurationServiceScalecube is created with a valid token
+    And   an existing repository containing an entry with key:X
+    And   user calls deleteEntry method by providing repository name and without any key
+    And   repository is deleted
+    When  user calls readList method with the name of the repository that was deleted
+    Then  user receives an error 'Configuration repository is not found'
+
+Scenario: createRepository() providing an existing repository name should return 'repositoryAlreadyExists' error
+  Given ConfigurationServiceScalecube with createRepository method
+  And   an existing repository
+  When  user calls createRepository method by providing the token and the existing name of repository
+  Then  user receives an error that the repository already exists
+
+Scenario: createEntry() with a key that exists in the repository
+  Given configurationServiceScalecube with createEntry method
+  And   an existing repository
+  And   an entry with key: X and value: Y that is existing in the repository
+  When  user calls createEntry method with key: X and value: Y and the name of this repository
+  Then  user receives an error //to confirm
+
+Scenario: Call updateEntry() with a key that is not in the repository
+  Given configurationServiceScalecube with updateEntry method
+  And   an existing repository
+  And   an entry with key: X and value: Y that is not existing in the repository
+  When  user calls updateEntry method with key: X and value: Y and the name of this repository
+  Then  user receives an error 'Configuration repository key not found'
+
+Scenario: Call readEntry() with invalid version
```

Comment:
May we ask Pavlo, to check "version" field if it has some restrictions (which values might be considered invalid)

Reply:
we deleted the scenario